autoquran/src/main/java/com/quran/labs/autoquran/common/SurahUtils.kt

```diff
@@ -0,0 +1,39 @@
+package com.quran.labs.autoquran.common
+
+import android.content.Context
+import android.text.TextUtils
+import com.quran.data.core.QuranConstants
+import com.quran.labs.autoquran.R
+
+fun getSuraName(
```

Comment:
are we able to use the method in `QuranDisplayData`?

Reply:
It requires more work to use this and have to move all the surah_name and stuff. i believe this would be much better in different PR.